app/templates/components/session-button.hbs

```diff
@@ -1,19 +1,14 @@
 {{#if this.session}}
-  <PaperMenu @offset="5 43" as |menu|>
-    <menu.trigger class="user-menu">
-      <img class="face" src= {{ this.session.avatarURL }} />
-      <Button> {{ this.session.nickName }} </Button>
-    </menu.trigger>
-    <menu.content as |content|>
-      <content.menu-item>
-        <Button @onClick={{action "changeUser"}}> {{t "components.sessionButton.changeUser"}} </Button>
-      </content.menu-item>
-
```

Comment:
Aca tome la correcta decisión de borrar el boton de Cambiar Usuario:
![image](https://user-images.githubusercontent.com/37090248/211909917-7ecbcca7-7cd7-4594-8bff-739a64758012.png)

Estaba teniendo problemas con este boton, porque el menu no desaparece al aparecer el modal de login, lo cual hace que no te deje clickear en ningun lado porque hay un div invisible cubriendo toda la pantalla:
![kobeni](https://user-images.githubusercontent.com/37090248/211912847-a0083187-1990-42e3-b92b-637359fb3652.gif)


La solucion a esto deberia ser manualmente llamar a `toggleMenu` al hacer click a este boton, pero la linea `this.storage.saveUser(null)` del metodo:
https://github.com/Program-AR/pilas-bloques/blob/ede45f549abc53b5d1d769f9daf762d6c33f76b9/app/services/pilas-bloques-api.js#L81-L86
Provoca que el menu no se cierre aunque se llame a `toggleMenu`. Intente poniendo el `toggleMenu` antes y despues de que se llame a esa linea. Comentando esa linea el `toggleMenu` si cierra el menu. Esto tiene mucho sentido y las razones por las que esto ocurre son:
‎
‎
‎
‎‎
Previendo que este iba a ser uno de "Esos" problemas, decidi solo borrar el boton, perdon rasta. Considerando que es solo un logout que ahorra un refresco de pagina y un entero click, creo que pilas bloques puede sobrevivir.

Reply:
¿Puede ser que falte borrar la action changeUser del js ahora que no se usa más? Y no sé si sus atributos derivados...